### PR TITLE
task-5

### PR DESCRIPTION
**Pull request by lexx129**
### Task-5
- implemented **importProductsFile** lambda + handler
- implemented **importFileParser** lambda + handler
- **importProductsFile** lambda function returns a correct response which can be used to upload a file into the S3 bucket
- Frontend application is integrated with importProductsFile lambda ([link]( https://d1amechee18gmk.cloudfront.net/))
(link to API: https://zfhk5qobji.execute-api.eu-west-1.amazonaws.com/dev/
Endpoint to test file uploading is 'GET /import')

Additional
- async/await is used in lambda handlers
- after imported file is processed, it's removed from 'uploaded' folder
- unit-tests for `importProductsFile` are added

### CODE REVIEW

**Comment by mmarkelov** [comment]
@lexx129 it's OK for me. I'll be waiting for tests

**Comment by lexx129** [comment]
> @lexx129 it's OK for me. I'll be waiting for tests

@mmarkelov, I've added some tests

File: product-service-ts/serverless.ts [hunk 1]
```diff
@@ -75,6 +75,36 @@ const serverlessConfiguration: Serverless = {
           }
         }
       ]
+    },
+    importProductsFile1: {
```
Comment (mmarkelov) [inline product-service-ts/serverless.ts]: I think `importProductsFile` will be better

File: product-service-ts/serverless.ts [hunk 2]
```diff
@@ -75,6 +75,36 @@ const serverlessConfiguration: Serverless = {
...
+        {
+          http: {
+            method: 'get',
+            path: 'import',
+            cors: true
+          }
+        }
+      ]
+    },
+    importFileParser2: {
```
Comment (mmarkelov) [inline product-service-ts/serverless.ts]: `importFileParser` will be better

**Comment by mmarkelov** [comment]
Also you can reduce your bundle size with this or you can move `aws-sdk` to **devDependencies**:
```js
// serverless.ts
...
custom: {
    webpack: {
      webpackConfig: './webpack.config.js',
      includeModules: {
        forceExclude: ['aws-sdk']
      }
    }
  },
```

**Comment by PavelZaletski** [comment]
One of the task requirements was:
Create a **new** service, called import-service, with **its own** serverless.yml file at the same level as product service.
The backend project structure should look like:
src/product-service
src/import-service

But you combined 2 services into one.

**Pull request merged**